frontend/src/routes/MediaRating/index.tsx

```diff
@@ -1,70 +1,92 @@
 import { Grid, GridItem } from '@chakra-ui/react'
 import { useEffect, useRef, useState } from 'react'
-import api from '../../api';
-
+import { PostTrialsRequest, PostTrialsRequestTrialMetadata, PostTrialsRequestRatingSet, PostTrialsRequestRatingSetRatingInner, PostTrialsRequestRatingSetRatingInnerEmotion } from "../../schema";
 import MediaDisplay from "../../features/MediaDisplay"
 import RatingSlider from "../../features/RatingSlider"
-import { ListImagesInner } from '../../schema'
 import "./MediaRating.css"
 import { useLocation, useNavigate } from 'react-router-dom'
-
-interface MediaRate {
-    mediaID: number
-    valence: number
-    arousal: number
-}
+import MediaList from "../../../src/imageFileNames.json"
 
 interface RatingPageProps {
     mediaType: string
 }
 
+const detasetName = "OASIS";
+const experimentLocation = "KC111";
+const platform = "";
+const rating_second_by_media = 0;
 const rateSeconds = 5;
-const ratingResult: MediaRate[] = [];
+const ratingResult: PostTrialsRequestRatingSetRatingInner[] = [];
 const [rateMin, rateDefault, rateMax] = [1, 5, 9];
 const mediaNum = 3;
 
 // TODO: mediaTypeに応じて、利用するエンドポイントを切り替える
 function RatingPage(props: RatingPageProps) {
     const [mediaSrc, setMediaSrc] = useState('');
     const mediaIndexRef = useRef<number>(0);
-    const listImages = useRef<ListImagesInner[]>();
     const valenceRef = useRef<number>(rateDefault);
     const arousalRef = useRef<number>(rateDefault);
+    const navigate = useNavigate();
+    const location = useLocation();
+    const subjectMetadata = location.state;
+    const [sliderValueTop, setSliderValueTop] = useState(50);
+    const [sliderValueBottom, setSliderValueBottom] = useState(50);
+    const mediaBaseSrc = "./images/oasis/";
+    const [started_at, setStarted_at] = useState("");
```

Comment:
こんな感じで一旦letで初期化しておいて、1枚目の画像表示前に再代入したら良いと思う。
```suggestion
let startedAt = "";
```